src/main/kotlin/no/nav/familie/ef/søknad/mapper/FeltMapperUtil.kt

```diff
@@ -26,3 +26,6 @@ fun dokumentfelt(dokumentnavn: String, vedleggMap: Map<String, List<Vedlegg>>):
         }
     }
 }
+
+
+fun String.tilHeltall(): Int = this.toDouble().toInt()
```

Comment:
En test for denne hadde vært fint. Som då dokumenterer att den ikke avrunder kun ned. Så 3.9 blir 3